### ISSUE TITLE
Display aliases and group names alongside the entityId and group id 

### ISSUE THREAD

**Issue by ghost** [hashicorp/vault#15740]
**Is your feature request related to a problem? Please describe.**
Its frustrating to see pages of ids when there are Names/Aliases available. It then requires opening a new tab and finding the id you want and coming back to the original page OR clicking through each id to figure out what it is.
Specifically:
1. the Group Members page only displays member IDs and doesn't display aliases.
2. The Entity > Groups page only displays group ids without a name

**Describe the solution you'd like**
1. Display the entity aliases on the Access > Group > Members page like they are displayed most other places
2. Display the group name on the Entities > $ENTITY > Groups page

**Describe alternatives you've considered**

**Explain any additional use-cases**
If there are any use-cases that would help us understand the use/need/value please share them as they can help us decide on acceptance and prioritization.

**Additional context**
Screenshots of the pages as they are now:
![Screenshot 2022-06-01 3 31 17 PM](https://user-images.githubusercontent.com/88346193/171488051-300a53e7-f3b9-4f92-a594-edabba73a5b6.png)
![Screenshot 2022-06-01 3 45 09 PM](https://user-images.githubusercontent.com/88346193/171488944-b1d82f5a-c4ad-4512-9cf2-b8bf30a6a735.png)

**Comment by twunderlich-grapl** [comment]
Aliases are assigned to the members, but they're not displayed in these specific views, but are displayed in other views. For example, if you go to edit a group, the members are displayed with their alias and id. See screenshot below

![Screenshot 2022-06-08 10 53 05 AM](https://user-images.githubusercontent.com/88346193/172648689-8ede1cef-e656-4710-9d37-922b2d8e84fa.png)


Note: I'm using HCP Vault, so the open source version may be different

**Comment by maxb** [comment]
I concur, this is a long standing frustration with these pages of the Vault UI for me too, it makes them basically useless for understanding current configs, and would be a valuable feature gap to fill.

**Comment by hashishaw** [comment]
Hi folks -- I completely understand the frustration here! Unfortunately we are displaying everything we get from the API on the members and groups pages. As an illustration, here is an example of the same `read` request we do on that page: 

```bash
vault read identity/group/id/0c2cb144-b069-8918-3032-1854531838a2
Key                  Value
---                  -----
alias                map[]
creation_time        2024-02-09T20:08:37.523029Z
id                   0c2cb144-b069-8918-3032-1854531838a2
last_update_time     2024-02-09T20:08:37.523029Z
member_entity_ids    [eda0eeaa-ea22-bb77-6fc7-476b0466ea56]
member_group_ids     <nil>
metadata             map[foo:bar]
modify_index         1
name                 some-group
namespace_id         oQqkj
parent_group_ids     <nil>
policies             [some-policy]
type                 internal
```

It is possible for the UI to do more API calls for the entities or groups in the response, but I'm wondering if that would be truly desired in terms of both performance and auditing. Another solution would be an update to the backend response that includes other information about the groups and entities. Genuinely interested in the community's thoughts around this! Thanks in advance.

**Comment by Eregrith** [comment]
@hashishaw Yes, it would be very much appreciated to have the names of groups on the Groups tab of an entity and the names of entities in the Members tab of a group ! Without these, it is completely useless to list the groups / entities: Why would anyone look for a group by ID on a web ui? The basic need I see would be when some dev leaves a project we go to the project's group > members > find the dev by its name on that page and remove it with the context menu on the right.

I doubt the performance overhead would be a problem.

**Comment by Eregrith** [comment]
We are almost one year later and there hasn't been a single touch on that topic. How are people managing the admin of the vault when pages are this useless in their display of information?
